opg-base-1604/Makefile

```diff
@@ -1,9 +1,20 @@
 .PHONY: build
+SHELL = '/bin/bash'
 
+mkfile_path := $(abspath $(lastword $(MAKEFILE_LIST)))
+current_dir := $(notdir $(patsubst %/,%,$(dir $(mkfile_path))))
+image := $(current_dir)
 registryUrl ?= registry.service.opg.digital
-
-image = opg-base-1604
+SEED := $(shell /bin/bash -c "echo $$RANDOM")
 
 build:
-	docker build $(no-cache) -t "$(registryUrl)/$(image)" .
-	# docker tag "$(registryUrl)/$(image)" "$(registryUrl)/$(image):$(newtag)"
+	docker-compose -f tests/docker-compose.yml build
+
+test:
+	echo "Seed: ${SEED}"
+	docker-compose -p "${SEED}" -f tests/docker-compose.yml up -d
+	inspec exec tests -t docker://${SEED}_basegen_1
+	docker-compose -p "${SEED}" -f tests/docker-compose.yml down
```

Comment:
recommended to have `down -v` this will remove any volumes made. it's not for now, as I see there's no volumes used, but it's a good failsafe to have in the event we eventually add some volumes.

Without -v then volumes will be left behind.

Reply:
When/if we have volumes, this is worth doing right now it's just more switches to maintain. 

Worth point out though, thanks.